mnefun/_mnefun.py

```diff
@@ -1772,3 +1777,28 @@ def gen_html_report(p, subjects, structurals, raw=True, evoked=True,
                             pattern=patterns)
         report_fname = get_report_fnames(p, subj)[0]
         report.save(report_fname, open_browser=False, overwrite=True)
+
+
+def plot_raw_psd(p, subjects, tmin=0, fmin=2, n_fft=2048, n_jobs=1):
+    """Plot data power for all available raw data files for a subject"""
+    for subj in subjects:
+        for file_type in ['raw', 'sss']:
+            fname = get_raw_fnames(p, subj, which=file_type, erm=False)
+            if len(fname) < 1:
+                raise Warning('Unable to find %s data file.' % file_type)
+            raw = io.Raw(fname, preload=True, allow_maxshield=True)
+            raw.plot_psd(tmin=tmin, tmax=raw.times[-1], fmin=fmin,
+                         fmax=raw.info['lowpass'] + 50, n_fft=n_fft,
+                         n_jobs=p.n_jobs, proj=False, ax=None, color=(0, 0, 1),
+                         picks=None)
+            plt.savefig(fname[0][:-4] + '_psd.png')
+        pca_file = get_raw_fnames(p, subj, which='pca', erm=False)
+        if len(pca_file) < 1:
+            raise Warning('Unable to find pca data file.')
+        raw = io.Raw(pca_file, preload=True, allow_maxshield=True)
+        raw.plot_psd(tmin=tmin, tmax=raw.times[-1], fmin=fmin,
+                     fmax=p.lp_cut, n_fft=n_fft,
+                     n_jobs=p.n_jobs, proj=False, ax=None, color=(0, 0, 1),
+                     picks=None)
+        plt.savefig(pca_file[0][:-4] + '_psd.png')
```

Comment:
This is not DRY. Why not just do `file_type in ['raw', 'sss', 'pca']` in the loop above?